Library/Formula/libantlr3c.rb

```diff
@@ -14,8 +15,31 @@ class Libantlr3c < Formula
   end
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
-    system "make install"
+    args = ["--disable-dependency-tracking",
+            "--disable-antlrdebug",
+            "--prefix=#{prefix}"]
+    args << "--enable-64bit" if MacOS.prefer_64_bit?
+    system "./configure", *args
+    if build.with? "exceptions"
+      inreplace "Makefile" do |s|
+        cflags = s.get_make_var "CFLAGS"
+        s.change_make_var! "CFLAGS", (cflags << " -fexceptions")
```

Comment:
My guess is that it was written to be a C library, and this is a feature that is only applicable to some C++ clients.